### DISCUSSION TITLE
Dark Mode on Companion Bar

### DISCUSSION THREAD

**Discussion by aldoyh** [dailydotdev/daily#802] [Feature Requests]
Hello,

I'd like to suggest adding the functionality of turning the articles on every other website you publish into Dark Mode, something like [this](https://chrome.google.com/webstore/detail/dark-mode-for-chrome/pjbgfifennfhnbkhoidkdchbflppjncb) Chrome Extension does.

I am actually using couple of those extensions, but hoped you'd help git rid of them 😃 

Let's Save our Retinas 👀 

Thanks.

**Comment by idoshamun** [comment]
Hi @aldoyh,

This is an interesting suggestion no doubt!
Thank you :)
